### ISSUE TITLE
Include images by name, not path

### ISSUE THREAD

**Issue by elgor** [latex4ei/tex4tum#6]
Instead of the path (e.g. ./img/rc.svg) like in
```
<object data="./img/rc.svg" type="image/svg+xml"></object>
```
images should be included only by an unique identifier:
```
<object data="rc_circuit_diagram" type="image/svg+xml"></object>
```

Advantage: independence from image location and extension

**Comment by hofbi** [comment]
Use the new Jekyll image include template for this issue

- [ ] `figure.html` for figures with description etx.
- [ ] `inline-image.html` for image include in eg. tables etc.

**Comment by elgor** [comment]
New problem: some svg images have dependencies to js files. Paths are hard-coded. 
Is there any possibility to specify global search paths of certain resources?

**Comment by hofbi** [comment]
Not sure, but if we put all images and svgs into the image folder it should be ok. But maybe as enhancement it would be nice

**Comment by hofbi** [comment]
resolved with 2363236899191c14cab05f07ed2cebd0e25c41bb

Additionally we could also write a similar liquid tag for the figure include, but for now its ok
